tests/README.md

```diff
@@ -41,6 +41,8 @@ To install MySQL follow the instructions provided [here](https://dev.mysql.com/d
 
 To install the WP test suite you need to first run `composer install` and `npm install` in the top level directory. Then you need to use the `tests/bin/install-wp-tests.sh` script to install the WP test suite.
 
+`install-wp-tests.sh` script needs `svn` to be installed in order to create test files, you can do that by running `npm install svn`
```

Comment:
Nice! Maybe add a period in the end? :)

Reply:
Good catch!

Reply:
Hey, let me know if I'm missing something, but I think we shouldn't suggest installing the npm svn package. The wp install shell script needs only the svn binary, so maybe we just provide a link to a multi-os svn install guide [like this one](https://subversion.apache.org/packages.html)?

Reply:
I totally wanted to write brew install

Reply:
Hey, I wanted to write brew install here.......

Reply:
Yeah, figured it might be a typo. 🙂 Looks good now!